spec/lib/pickup_bot/commands/set_time_spec.rb

```diff
@@ -32,9 +32,30 @@
   end
 
   context "an active game exists" do
+    before do
+      Game.create(chat_id: fake_chat_id, required_players: 5)
+    end
+
+    scenario "new player joins game, a new player is created" do
+      message = Telegram::Bot::Types::Message.new(message_params('/set_time     06:00 '))
+
+      pickup_bot.run(message)
+
+      expect(a_request(:post, "https://api.telegram.org/botfake-token/sendMessage").
+              with(body: {
+                "chat_id" => "123",
+                "text" => I18n.t(
+                  "bot.time",
+                  username: user_params[:username],
+                  time: "06:00"
+                )
+              }
+            )).to have_been_made.times(1)
+      expect(Game.last.time).to eq "06:00"
+    end
+
     scenario "new player joins game, a new player is created" do
```

Comment:
weird scenario title?

Reply:
Yeah was copied, still need to change that